components/membership/registration-impl/src/main/kotlin/net/corda/membership/impl/registration/dynamic/mgm/MGMRegistrationMemberInfoHandler.kt

```
@@ -78,9 +79,9 @@ internal class MGMRegistrationMemberInfoHandler(
 
     @Suppress("ThrowsCount")
     private fun getKeyFromId(keyId: String, tenantId: String, expectedCategory: String): PublicKey {
-        return cryptoOpsClient.lookup(
+        return cryptoOpsClient.lookupKeysByIds(
             tenantId,
-            listOf(keyId)
+            listOf(ShortHash.of(keyId))
```

If the short hash is invalid we need to throw the correct exception